src/extension.ts

```diff
@@ -51,27 +65,48 @@ export function activate(context: vscode.ExtensionContext) {
     l2Version === undefined ||
     semver.lt(l2Version, MIN_VERSION_TO_CHECK)
   ) {
-    // The L2 version, if less than v1.5.0, does not support the `l2 -e <filepath>` feature.
-    // Therefore, we use the extension to fetch variables from the l2.env file for suggestions.
+    /**
+     * The L2 version, if less than v1.5.0, does not support the `l2 -e <filepath>` feature.
```

Comment:
Let's ditch this feature. We'll go 100% LSP from v1.5.9. We can remove it from backend in `v1.5.9` as well.

Reply:
Done